<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTrainingWindow</class>
 <widget class="QDialog" name="AddTrainingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter un entrainement personnalis√©</string>
  </property>
  <widget class="QPushButton" name="validateBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="fileLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="fileChooseBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>42</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Choisir un fichier d'entrainement (.udk)</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color : red;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="faqBtn">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>190</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Besoin d'aide ?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddTrainingWindow</receiver>
   <slot>hideAddTrainingWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileChooseBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddTrainingWindow</receiver>
   <slot>askFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddTrainingWindow</receiver>
   <slot>onNameChange(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>validateBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddTrainingWindow</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>faqBtn</sender>
   <signal>clicked()</signal>
   <receiver>AddTrainingWindow</receiver>
   <slot>showFaq()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>hideAddTrainingWindow()</slot>
  <slot>askFile()</slot>
  <slot>onNameChange(QString)</slot>
  <slot>add()</slot>
  <slot>showFaq()</slot>
 </slots>
</ui>
